specification/cognitiveservices/data-plane/ContentModerator/readme.md

```diff
@@ -51,18 +51,18 @@ python:
   license-header: MICROSOFT_MIT_NO_VERSION
   add-credentials: true
   payload-flattening-threshold: 2
-  namespace: azure.cognitiveservices.vision.contentmoderator
-  package-name: azure-cognitiveservices-vision-contentmoderator
+  namespace: azure.cognitiveservices.contentmoderator
```

Comment:
As discussed in the email thread, advertising of Content Moderator is under Vision. We need a clear justification to do not follow the official branding for this.
Not too that this package is already plugged in the Python repo in the automatic update system:
https://github.com/Azure/azure-sdk-for-python/tree/master/azure-cognitiveservices-vision-contentmoderator

so changing this breaks the SDK repo. Even if finally we rename the package, this PR is not the good place and should be done in sync with me (Python SDK owner and published)